Application/Interfaces/ITraderManagement.cs

```diff
@@ -0,0 +1,15 @@
+using Domain.Entities;
+
+namespace Application.Interfaces
+{
+
+    public interface ITraderManagement
```

Comment:
change the name of this interface to ITraderService, this is a convention when using c#